Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,52 @@
+// Task #1 #2
+let a = 1;
+console.log(typeof a); // expected output: number
+let aNumber = new Number(1);
+console.log(typeof aNumber); // expected output: object
+let b = 'abc';
+console.log(typeof b); // expected output: string
+let bString = new String('abc');
+console.log(typeof bString); // expected output: object
+let c = true;
+console.log(typeof c); // expected output: boolean
+let cBoolean = new Boolean(true);
+console.log(typeof cBoolean); // expected output: object
+let d = { da: 1, db: 2 };
+console.log(typeof d); // expected output: object
+let dObject = new Object('{ da: 1, db: 2 }');
+console.log(typeof dObject); // expected output: object
```

Comment:
Not an object but a new string will be created:

![image](https://user-images.githubusercontent.com/8523446/66717078-be6ed280-eddd-11e9-920c-d77663454b24.png)

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object

Reply:
Disscussed in person: decided to leave as it is.
console.log(typeof dObject); shows Object, but dObject is a String.